cet/cet_tests.sh

```diff
@@ -3,7 +3,16 @@
 # Copyright (c) 2022 Intel Corporation
 # @Desc  Test script to verify Intel CET functionality
 
-cd "$(dirname $0)" 2>/dev/null; source ../.env
+cd "$(dirname "$0")" 2>/dev/null && source ../.env
+
+readonly NULL="null"
+readonly CONTAIN="contain"
+
+TEST_MOD="cet_ioctl"
+TEST_MOD_KO="${TEST_MOD}.ko"
+KO_FILE="../cet_driver/${TEST_MOD_KO}"
+
+teardown_handler="cet_teardown"
```

Comment:
> Again we met this issue. Do we have rule to ignore this kind of issue, or do we plan to fix it?
> 
> cet/cet_tests.sh:15 - teardown_handler appears unused. Verify use (or export if used externally)., refer to https://github.com/koalaman/shellcheck/wiki/SC2034

Actually  I thought it's a global variable "teardown_handler" defined in common.sh  and will be used in other scripts.
It will not imact the usage.

Anyway, yes, we could use export to fix the shellcheck warning.
Thanks for suggestion!

Thanks!

Reply:
The warning is fixed.

Thanks!